libs/api/api_internal.h

```diff
@@ -41,15 +41,14 @@ typedef struct _ebpf_object
     std::vector<ebpf_map_t*> maps;
 } ebpf_object_t;
 
-uint32_t
-ebpf_get_program_byte_code(
-    const char* file_name,
-    const char* section_name,
+_Return_type_success_(return == ERROR_SUCCESS) uint32_t ebpf_get_program_byte_code(
```

Comment:
https://docs.microsoft.com/en-us/cpp/code-quality/annotating-function-behavior?view=msvc-160 says that `_Return_type_success_` is for annotating typedefs (since they can be used as return types and other things like arguments and struct members) not functions. `_Success_` is what should instead be used on function annotation.

However a better solution would be to make this API return `ebpf_result_t` and make the `ebpf_result_t` have the `_Return_type_success_` annotation.